common/app/views/main.scala.html

```diff
@@ -4,11 +4,66 @@
 @import views.support.Commercial.topBelowNavSlot
 @import views.support.{Commercial, RenderClasses}
 @import model.Page.getContent
+@import model.GalleryPage
+
+@headerAndTopAds(showAdverts: Boolean, edition: Edition, adBelowNav: Boolean) = {
+    @if(!(page.metadata.shouldHideHeaderAndTopAds && getContent(page).exists(_.tags.isArticle))) {
+        @if(showAdverts) {
+            @fragments.commercial.topBanner(page.metadata, edition)
+        }
+
+        @fragments.header(page)
+
+        @if(adBelowNav) {
+            @fragments.commercial.topBannerBelowNav()
+        }
+
+        @if(showAdverts) {
+            @fragments.commercial.topBannerMobile(page.metadata, edition)
+        }
+
+        <div id="maincontent" tabindex="0"></div>
+    }
+}
+
+@immersiveMainMedia(page: GalleryPage) = {
+    <div class="new-gallery content tonal--tone-media immersive-main-media">
+        @defining(
+            page.item.content.trail.trailPicture.head
+        ) { picture =>
+            <img src="http:@Html(ImgSrc.findNearestSrc(picture, Profile(width = Some(2200))).getOrElse(""))"
```

Comment:
The `src` is incorrect here. This will soon be replaced with a `picture` element